.github/workflows/publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      # https://github.com/actions/virtual-environments/issues/937
+      - name: Temporal fix
```

Comment:
Is this "temporal", as in, a fix related to the flow of time, or "temporary", as in, not going to last very long?